.github/workflows/go-i18n.yml

```diff
@@ -0,0 +1,35 @@
+name: Go i18n
```

Comment:
avoid creating new workflows, write new jobs in existing ones instead

Reply:
That was for having a custom path trigger, this workflows need to be runned only if some touches `*/i18n.go`.